docs/PRIORITIES.md

```diff
@@ -1,46 +1,35 @@
 # Post-0.25/GoS Pre-Release
 
-## Staking/Slashing/Stability
-
-- Other slashing issues blocking for launch - [#1256](https://github.com/cosmos/cosmos-sdk/issues/1256)
```

Comment:
These aren't done, e.g. https://github.com/cosmos/cosmos-sdk/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astaking+label%3Aprelaunch-2.0

Reply:
Ah well the link there earlier listed no slashing issues. Will update.